### PR TITLE
queries for methods

### PR DESCRIPTION
**Pull request by PiyushMishra**

### CODE REVIEW

File: resources/mappingForNewMethodIndex.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+{  
+   "custom":{  
+      "properties":{  
+         "file":{  
+            "type":"string",
+            "index":"no"
+         },
+         "methodTokens":{
```
Comment (ScrapCodes) [inline resources/mappingForNewMethodIndex.md]: Can you add line numbers under this ? and remove "tokens" fully ?

**Comment by PiyushMishra** [comment]
Now we would have a new index called *importsmethods*.

File: bin/upload_to_es.sh [hunk 2]
```diff
@@ -20,16 +20,17 @@
 # DISCLAIMER: There are no defensive checks please use it carefully.
 
 echo "Clearing betterdocs related indices from elasticsearch."
-curl -XDELETE 'http://localhost:9201/betterdocs/'
-curl -XDELETE 'http://localhost:9201/sourcefile/'
-curl -XDELETE 'http://localhost:9201/repository/'
+curl -XDELETE 'http://localhost:9200/betterdocs/'
+curl -XDELETE 'http://localhost:9200/sourcefile/'
+curl -XDELETE 'http://localhost:9200/repository/'
+curl -XDELETE 'http://localhost:9200/importsmethods/'
```
Comment (ScrapCodes) [inline bin/upload_to_es.sh]: Why did you change it to 9200 ? Please correct it to 9201.
